setup.py

```diff
@@ -45,6 +47,15 @@
 if not gin_config_file_local.exists():
     copy(src=gin_config_file_base, dst=gin_config_file_local)
 
+# Bug related to sonpy on M1 Mac being installed but not running properly
+if sys.platform == "darwin" and platform.processor() == "arm":
```

Comment:
For my own curiosity, why would the difference between `platform.processor()` and `plataform.machine()` would be?

Reply:
On M1:

`platform.machine() = "arm64"`
`platform.processor() = "arm64"`

On my Windows, the `platform.processor()` is much more verbose about model number and specs from Intel, and machine is just `"AMD64"`

So if we ever need to extend, possibly `machine` might generalize cleaner but its similar to the older conversations about `sys.platform` syntax vs. all the other ways of getting string representations of platforms

I do prefer the .toml language however for handling all of that

Reply:
Thanks, I just tested and they are the same in my system:

```python
In [2]: import platform

In [3]: platform.machine()
Out[3]: 'x86_64'

In [4]: platform.processor()
Out[4]: 'x86_64'
```